pyproject.toml

```diff
@@ -42,6 +42,7 @@ vertexai = ["google-cloud-aiplatform >= 1.38.0"]
 together = ["together"]
 argilla = ["argilla >= 1.22.0", "sentence-transformers >= 2.0.0", "textdescriptives >= 2.0.0"]
```

Comment:
Shouldn't we pin Argilla to a higher version here?

Reply:
Is there a higher version than `argilla>=1.22` right now?

Reply:
No, I mean when we merge the `httpx` bumped version